src/test/java/io/aiven/kafka/connect/gcs/testutils/BucketAccessor.java

```diff
@@ -191,6 +191,13 @@ private InputStream getDecompressedStream(final InputStream inputStream, final S
         }
     }
 
+    public final List<String> readJsonLLines(final String blobName,
+                                             final String compression) {
+        Objects.requireNonNull(blobName, "blobName cannot be null");
+        return readLines(blobName, compression).stream()
+                .collect(Collectors.toList());
+    }
+
```

Comment:
Doesn't seem to be used. Also, doesn't seem to add much value on top of `readLines`.

Reply:
- [ ] Remove readJsonLines